### ISSUE TITLE
Wrong indentation of comments when semicolons are missing

### ISSUE THREAD

**Issue by adrianton3** [beautify-web/js-beautify#1083]
# Input
The code looked like this before beautification:
```
f()
/* a */
```

# Expected Output
The code should have looked like this after beautification:
```
f()
/* a */
```

# Actual Output
The  code actually looked like this after beautification:
```
f()
    /* a */
```

# Steps to Reproduce
1. Open http://jsbeautifier.org/
2. Paste the code
3. Hit the beautify button
4. Observe the misalignment of `/* a */`
5. Add a `;` for `f()`
6. Hit beautify again
7. Observe the correct alignment of `/* a */`

## Environment
OS: N/A

## Settings
The default settings for http://jsbeautifier.org/
This is the options argument supplied to the `js_beautify` function:

```json
{
"brace_style": "collapse",
"break_chained_methods": false,
"comma_first": false,
"e4x": false,
"end_with_newline": false,
"indent_char": " ",
"indent_inner_html": false,
"indent_scripts": "normal",
"indent_size": "4",
"jslint_happy": false,
"keep_array_indentation": false,
"max_preserve_newlines": "5",
"preserve_newlines": true,
"space_before_conditional": true,
"unescape_strings": false,
"wrap_line_length": "0"
}
```

**Comment by tobias-zucali** [comment]
There is a missing semicolon. Does it work with this?
f();
/* a */

**Comment by adrianton3** [comment]
Yes it does (see the title and repro steps), but I'd expect the indentation to be the same. Semicolons shouldn't affect indentation since they're optional. I don't re-indent my code when I add/remove `;` from my code.
